docker/dev.Dockerfile

```diff
@@ -27,7 +27,9 @@ RUN apt-get update --fix-missing && \
         python3.11-distutils \
         python3.12 \
         python3.12-dev \
-        python3.12-distutils && \
+        python3.12-distutils \
+        python3.13 \
+        python3.13-dev && \
```

Comment:
is -distutils not here intentionally?

Reply:
Yes, distutils was removed from Python 3.12. But Ubuntu/Debian provided a package for Python 3.12. There's no such package for Python 3.13 and it isn't required for building things anymore on Python 3.13.